libs/core/src/lib/fixed-card-layout/fixed-card-layout.component.ts

```diff
@@ -47,14 +66,16 @@ type CardColumn = CardDefinitionDirective[];
     selector: 'fd-fixed-card-layout',
     templateUrl: './fixed-card-layout.component.html',
     styleUrls: ['./fixed-card-layout.component.scss'],
-    changeDetection: ChangeDetectionStrategy.OnPush,
-    encapsulation: ViewEncapsulation.None
+    changeDetection: ChangeDetectionStrategy.OnPush
```

Comment:
Is there any reason of adding styles encapsulation?

Reply:
i have removed styles encapsulation, this fixes tabing issue on content-area of card.
Because of styles encapsulation content area outer border was not getting focused as content is not emitting focus.